engine/apps/slack/representatives/alert_group_representative.py

```diff
@@ -278,10 +278,9 @@ def on_ack_reminder_triggered(self):
         step = AcknowledgeConfirmationStep(self.log_record.alert_group.channel.organization.slack_team_identity)
         step.process_signal(self.log_record)
 
-    def on_custom_button_triggered(self):
-        CustomButtonProcessStep = ScenarioStep.get_step("distribute_alerts", "CustomButtonProcessStep")
-        step = CustomButtonProcessStep(self.log_record.alert_group.channel.organization.slack_team_identity)
-        step.process_signal(self.log_record)
+    def on_custom_webhook_triggered(self):
+        # TODO: I think we need to fill in this logic?
```

Comment:
Should be able to just remove this. I believe it is left over from when custom button was a custom button defined to be pressed in slack.

Reply:
Thanks for that, addressed in https://github.com/grafana/oncall/pull/2544/commits/3b4762e4793ca805b651cfb662e945e141419175